spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/JdbcSessionProperties.java

```diff
@@ -42,7 +43,10 @@
 	 */
 	private String tableName = DEFAULT_TABLE_NAME;
 
-	private final Initializer initializer = new Initializer();
+	/**
+	 * Create the required Spring Session tables on startup.
```

Comment:
That removes silently a feature isn't it? It's probably for the better but I just want to double check.